assets/scripts/codeflask.js

```diff
@@ -166,7 +166,8 @@ CodeFlask.prototype.handleScroll = function(textarea, highlightPre) {
             this.scrollTop = roundedScroll;
         }
 
-        highlightPre.style.top = "-" + roundedScroll + "px";
+        // https://github.com/kazzkiq/CodeFlask.js/pull/41/files
+        highlightPre.scrollTop = roundedScroll;
```

Comment:
This gets around a style bug that occurs when the codeflask's text content has to horizontally scroll. The commented URL contains the PR with the prosed fix, but the need for this fix is also described here: https://github.com/kazzkiq/CodeFlask.js/issues/34